engine/src/main/java/com/datatorrent/stram/security/StramUserLogin.java

```diff
@@ -51,6 +51,7 @@
   public static final String DT_AUTH_PREFIX = StreamingApplication.DT_PREFIX + "authentication.";
   public static final String DT_AUTH_PRINCIPAL = DT_AUTH_PREFIX + "principal";
   public static final String DT_AUTH_KEYTAB = DT_AUTH_PREFIX + "keytab";
+  public static final String DT_APP_PATH_IMPERSONATED = DT_AUTH_PREFIX + "application.path.impersonated";
```

Comment:
It might be better to call this AUTH_PREFIX + ".impersonation.application.path" to emphasize impersonation. Sorry, I didn't notice this earlier.